cores/esp8266/Tone.cpp

```diff
@@ -35,8 +35,8 @@ static void _startTone(uint8_t _pin, uint32_t high, uint32_t low, unsigned long
 
   pinMode(_pin, OUTPUT);
 
-  high = std::max(high, (uint32_t)100);
-  low = std::max(low, (uint32_t)100);
+  high = std::max(high, (uint32_t)100);  // 5KHz maximum tone for Arduino compatibility
```

Comment:
Why not just set it to 25 here (i.e. 20Khz max)?  I think you're fine allowing a larger range than Arduino old-style core.

Reply:
I'm good with it as well.  It's in.